README.md

```diff
@@ -340,6 +340,7 @@ Currently the integration supports the following languages:
 - Norwegian (bokmål and nynorsk)
 - Russian
 - Portuguese (Portugal)
+- German (Germany)
```

Comment:
Can your please list these in alphabetic order 😄 I know it wasn't before, but just noticed that it wasn't